repository/CargoPackageManager-Tests/PBMetacelloProjectTest.class.st

```diff
@@ -1,45 +0,0 @@
-Class {
-	#name : #PBMetacelloProjectTest,
-	#superclass : #TestCase,
-	#category : 'CargoPackageManager-Tests-Model'
-}
-
-{ #category : #tests }
-PBMetacelloProjectTest >> testCanCreateAPBMetacelloProjectFromABaselineProjectDependency [
-	| project |
```

Comment:
why removing these tests?

Reply:
Because *I was mad* at the fact that I had to implement something that was supposed to be done two weeks ago, and these tests were
- testing no useful behaviour to me (just the fact that a data object can be created from another data object and that the good amount of setters were called??!!). These are not behaviour related tests.
- Since they are data-related tests, they are somehow hardcoding the internal structure of objects and making it more difficult to me to actually change that internal structure (which I had to do because otherwise metacello baseline loading and conflict detection couldn't work?)
- And they were broken because they are using ZnUrls and we should not use ZnUrls because even if we call them "urls", not all repository urls are urls.

Reply:
And also, this shows also:
- that I trusted you to finish the implementation (and actually merged the PR without looking it in deep). And that did not happen.
- and worse, I fell that you don't trust me, because It looks like you are actually reviewing EVERY pull request we are doing.

Reply:
I will not reply on this thread. It is not the place to discuss that. Neither writing is a good vector of communication for that. We can discuss at work tomorrow.
Just a fact: PR are there for reviews. If no one look at them, why to use them? It is also the best way to understand how the code evolves and ask questions on things we do not understand or get the reason.